app/models/spree/subscription.rb

```diff
@@ -97,10 +98,18 @@ def not_changeable?
       cancelled?
     end
 
+    def type_of_sub
+      if sub_type == 'Spree::Subscriptions::LabelStatus'
```

Comment:
I would use case statement here, but it's just my taste :D

Reply:
I also prefer using the case statement here, but it doesn't matter, cuz it seems this part will be also refactored as well if we want to remove all the 'Spree::Subscriptions::Period'/frequency from the code.